attest/verifier/src/ias.rs

```diff
@@ -104,7 +104,7 @@ impl IasReportVerifier {
         // will support
         let profile = Profile::new(
             vec![HashType::Sha256, HashType::Sha384, HashType::Sha512],
-            vec![PkType::Rsa, PkType::Ecdsa],
+            vec![PkType::Rsa, PkType::Eckey, PkType::Ecdsa],
```

Comment:
This had to be updated in order for MbedTls to handle the ECDSA certificates. The previous `PkType::Ecdsa` didn't fulfill the full need in order to work with elliptical curves.